ui-tests/src/test/java/org/openmrs/reference/UnidentifiedPatientKeyboardTest.java

```diff
@@ -20,7 +20,6 @@
  * Created by tomasz on 22.07.15.
  */
 public class UnidentifiedPatientKeyboardTest extends TestBase {
-    private HeaderPage headerPage;
```

Comment:
For quality assurance add an empty line to separate the class declaration from its declared variables. Did you get a chance to see some artifacts [here ](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/476#issuecomment-870471982)?